docs/amber_script.md

```diff
@@ -435,6 +438,27 @@ The following commands are all specified within the `PIPELINE` command.
   FRAMEBUFFER_SIZE _width_ _height_
 ```
 
+```groovy
+  # Require that subgroups must be launched with all invocations active
+  # for given shader. Only valid in the compute stage.
+  REQUIRE_FULL_SUBGROUPS {name_of_shader}
```

Comment:
Not a huge fan of this syntax for setting subgroup information. There seems to be 3 related things begin set here. So, a few questions. If these are always related to a given shader, shoudl these all be options on the `ATTACH` command?

Would this be better as a `SUBGROUP` block similar to `DEPTH` and `STENCIL`.

Reply:
Is this a per-shader item or a per-pipeline item, or shader/pipeline? Would you have one shader, in two pipelines with different subgroup information?

Reply:
It's a per-shader/pipeline item. It is valid to use the same shader in two pipelines with different subgroup settings.

Reply:
That syntax looks pretty good. I need to look at the details.

Reply:
I updated the syntax to match the form proposed in #852 (comment)